File: app/src/main/java/com/absoluteapps/arthurl/mobooru/Help.java
package com.absoluteapps.arthurl.mobooru;

import android.graphics.Color;
import android.graphics.PixelFormat;
import android.os.Bundle;
import android.support.design.widget.Snackbar;
import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

public class Help extends AppCompatActivity {

    private ViewPager mViewPager;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().setFormat(PixelFormat.TRANSLUCENT);
        setContentView(R.layout.help_page);

        mViewPager = (ViewPager) findViewById(R.id.viewpager);

        // Set an Adapter on the ViewPager
        mViewPager.setAdapter(new HelpAdapter(getSupportFragmentManager()));

        // Set a PageTransformer
        mViewPager.setPageTransformer(false, new HelpPageTransformer());

        Snackbar snackbar = Snackbar
                .make(mViewPager, "", Snackbar.LENGTH_INDEFINITE)
                .setAction("DONE", new View.OnClickListener() {
                    @Override
                    public void onClick(View view) {
                        finish();
                    }
                });
        snackbar.getView().setBackgroundColor(Color.parseColor("#cc434343"));
        snackbar.show();
    }

}

File: app/src/main/java/com/absoluteapps/arthurl/mobooru/Help.kt
package com.absoluteapps.arthurl.mobooru

import android.content.SharedPreferences
import android.graphics.Color
import android.graphics.PixelFormat
import android.os.Bundle
import android.preference.PreferenceManager
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.viewpager.widget.ViewPager
import com.google.android.material.snackbar.Snackbar

class Help : AppCompatActivity() {

    private var mViewPager: ViewPager? = null
    private lateinit var prefs: SharedPreferences
    private lateinit var prefsEditor: SharedPreferences.Editor

    override fun onCreate(savedInstanceState: Bundle?) {
        prefs = PreferenceManager.getDefaultSharedPreferences(baseContext)
        prefsEditor = prefs.edit()

        // Dark mode
        setTheme(if (prefs.getBoolean("DARK_MODE", false)) R.style.AppThemeDark else R.style.AppThemeLight)

        super.onCreate(savedInstanceState)
        window.setFormat(PixelFormat.TRANSLUCENT)
        setContentView(R.layout.help_page)

        mViewPager = findViewById<View>(R.id.viewpager) as ViewPager

        // Set an Adapter on the ViewPager
        mViewPager!!.adapter = HelpAdapter(supportFragmentManager)

        // Set a PageTransformer
        mViewPager!!.setPageTransformer(false, HelpPageTransformer())

        val snackbar = Snackbar
                .make(mViewPager!!, "", Snackbar.LENGTH_INDEFINITE)
                .setAction("DONE") { finish() }
        snackbar.view.setBackgroundColor(Color.parseColor("#cc434343"))
        snackbar.show()
    }

}
